nbs/explains/minidataapi.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
_Commented on notebook `nbs/explains/minidataapi.ipynb` Cell 23 Line 1_
```python
for user in users(where="year_started=2019"):
```
Let's use a named parameter here to demonstrate that. Also the spec includes a number of other params, so show those too. (see fastsql or fastlite for details)